docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -111,6 +111,20 @@ $ sudo sed -ri 's/.?group\s?=\s?".+"/group = "kvm"/1' /etc/libvirt/qemu.conf
 $ newgrp libvirt
 ----
 
+=== Start libvirtd service
```

Comment:
Apologies, I initially overlooked that we also need an anchor ID here:

```adoc
[[kvm-driver-libvirtd-service]]
=== Start libvirtd service
```

Reply:
Done.